src/Uno.UWP/Security/Authentication/Web/WebAuthenticationBrokerProvider.wasm.cs

```diff
@@ -11,7 +11,7 @@ namespace Uno.AuthenticationBroker
 {
 	partial class WebAuthenticationBrokerProvider
 	{
-		protected virtual string[] GetApplicationCustomSchemes()
```

Comment:
I'm assuming this is a safe change. Application code is compiled against the reference library which uses `IEnumerable<string>`, so my thought is that the return type change here can't be observed.

Reply:
All good, it's fixing a bug that may have already been there.